### DISCUSSION TITLE
[ajax] Rename "request" to "fetch"

### DISCUSSION THREAD

**Discussion by ruphin** [ing-bank/lion#1367] [Ideas]
We propose changing the `ajax.request()` and `ajax.requestJson()` API to `ajax.fetch()` and `ajax.fetchJson()`, to align more closely to platform naming conventions. This change will:

- Help users understand more intuitively that these have the same function signature as the [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API).
- Avoid confusion with [Request](https://developer.mozilla.org/en-US/docs/Web/API/Request) when discussing this API.

**Comment by ruphin** [comment]
I have prepared a branch with the suggested change here: https://github.com/ing-bank/lion/tree/chore/ajax_rename

**Reply by daKmoR** [reply]
change looks good - feel free to open a Pull Request once you think it's ready to be merged

**Comment by jorenbroekema** [comment]
https://github.com/ing-bank/lion/pull/1371 taken care of here :) thanks!
